Index: src/main/java/Field.java
===================================================================
diff --git a/src/main/java/Field.java b/src/main/java/Field.java
--- a/src/main/java/Field.java	
+++ b/src/main/java/Field.java	
@@ -3,8 +3,8 @@
 public class Field {
     private int[] fieldValue = {0, 250, -100, 100, -20, 180, 0, -70, 60, -80, -50, 650};
     String[] fieldName = {"start", "Tower", "Crater", "Palace gates", "Cold Desert", "Walled city", "Monastery", "Black cave", "Huts in the mountain", "The Werewall", "The pit", "Goldmine"};
-    private int player1Placement = 0;
-    private int player2Placement = 0;
+    public int player1Placement = 0;
+    public int player2Placement = 0;
 
 
     public void setPlayer1Placement(int player1Placement) {
